src/js/gi/components/profile/CalculatorForm.jsx

```diff
@@ -16,7 +16,6 @@ class CalculatorForm extends React.Component {
     this.renderScholarships = this.renderScholarships.bind(this);
     this.renderTuitionAssist = this.renderTuitionAssist.bind(this);
     this.renderEnrolled = this.renderEnrolled.bind(this);
```

Comment:
👍   

The one thing I would bind is `this.props.onShowModal`; if you set up that function in the constructor, you can avoid the performance impact of binding within render functions.